torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""CropHarvest datasets."""
+
+import glob
+import json
+import os
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import download_and_extract_archive, download_url
+
+
+class CropHarvest(NonGeoDataset):
+    """CropHarvest dataset.
+
+    The CropHarvest <https://github.com/nasaharvest/cropharvest> datataset is a
+    classification dataset.
+
+    Dataset features:
+
+    * _ pixel single timeseries with croptype labels
+    * 18 bands per image over 12 months
+
+    Dataset format:
+
+    * images are 12x18 ndarrays with 18 bands over 12 months
+
+    Dataset properties:
+
+    1. is_crop - whether or not a single pixel contains cropland
+    2. classification_label - optional field identifying a specific croptype
+    3. dataset - source datset for the imagery
+    4. lat
+    5. lon
+
+
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://openreview.net/forum?id=JtjzUXPEaCu
+
+    This dataset requires the following additional library to be installed:
+
+       * `h5py <https://pypi.org/project/h5py/>`_ to load the dataset
+    """
+
+    # *https://github.com/nasaharvest/cropharvest/blob/main/cropharvest/bands.py
+    all_bands = [
+        "VV",
+        "VH",
+        "B2",
+        "B3",
+        "B4",
+        "B5",
+        "B6",
+        "B7",
+        "B8",
+        "B8A",
+        "B9",
+        "B11",
+        "B12",
+        "temperature_2m",
+        "total_precipitation",
+        "elevation",
+        "slope",
+        "NDVI",
+    ]
+    rgb_bands = ["B11", "B8", "B2"]
```

Comment:
Normally we visualize Sentinel-2 imagery using Bands 4, 3, 2. Where did 11, 8, 2 come from?